src/fsharp/TastPickle.fs

```diff
@@ -1277,7 +1287,13 @@ let p_nonlocal_val_ref (nlv:NonLocalValOrMemberRef) st =
     p_bool pkey.MemberIsOverride st 
     p_string pkey.LogicalName st 
     p_int pkey.TotalArgCount st 
+    let isStructThisArgPos = 
+        match key.TypeForLinkage with 
```

Comment:
What does TypeForLinkage here mean?

Reply:
Like .NET metadata, pickled F# metadata contains references from one assembly to another. These must be "linked".  The "TypeForLinkage" is thus the signature of a method that might be overloaded, used to work out which method is being referred to in another assembly.